tensorflow/compiler/tests/qr_op_test.py

```diff
@@ -24,12 +24,17 @@
 import numpy as np
 
 from tensorflow.compiler.tests import xla_test
+from tensorflow.python.framework import test_util
 from tensorflow.python.ops import array_ops
 from tensorflow.python.ops import linalg_ops
 from tensorflow.python.ops import math_ops
 from tensorflow.python.platform import test
 
 
+@test_util.run_all_without_tensor_float_32(
+    "It's unknown why this test requires TF32 to be disabled")
+# TODO(reedwm): Determine why this test requires TF32 disabled. Debugging is
```

Comment:
Probably file a bug or GH issue (unless you intend to look at this immediately after this PR goes in)?

Reply:
I commented all the reasons why this test needs fp32 and filed an internal bug